server/target/target.go

```diff
@@ -678,7 +678,7 @@ func readPaginatedTargetsFromOLAPDB(ctx context.Context, env environment.Env, re
 	q.AddWhereInClause("commit_sha", outerCommitQuery)
 	q.AddWhereClause("group_id = ?", groupID)
 	q.AddWhereClause("repo_url = ?", repo)
-	q.SetOrderBy("label", true /*=ascending*/)
+	q.SetOrderBy("start_time_usec DESC, label", true /*=ascending*/)
```

Comment:
can you put start_time_usec DESC after label?

I think the table is "group_id, repo_url, commit_sha, label, invocation_uuid"

Reply:
not sure if it matters since the ORDER BY is being applied to an aggregation result? but I updated it anyway, seems less error prone to follow the index ordering :shrug: